apps/cli/src/node/docker-compose-host.yaml

```diff
@@ -22,3 +22,28 @@ services:
             ENABLE_SUNODO_NODE: true
             # advance-runner
             SNAPSHOT_ENABLED: false
+
+    traefik-config-generator:
```

Comment:
I see we are using the same service-name for all sub-composes to augment the environment, is that correct?

Reply:
yes, the idea was to keep the configuration close to the service it configures